news/2025-03-20-wasm-2.0/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Wasm 2.0 Completed”
+date: 2025-03-20
+layout: news
+---
+
+As of last December, release 2.0 of the Wasm specification is “official”!
+
+If you have been following the developments of the Wasm standard, then version 2.0 may sound like rather old news to you. And indeed, the Wasm Community and Working Groups had reached consensus and finished the specification in early 2022. All major implementations have been shipping 2.0 for even longer. But for a variety of non-technical reasons, it took a while for it to advance through the W3C process and reach the status of [“Recommendation Draft”](https://www.w3.org/TR/wasm-core-2/).
```

Comment:
I think I would call the status "Candidate Recommendation" (leaving out the distinction between the "Snapshot" and "Draft" variants). That seems to be in more common usage.

Reply:
Yeah, I consciously used "Draft" because that's what the document header prominently says. If I don't include it here, people might get the impression that the document they are looking at is not the one I'm talking about but some pre version.

Reply:
True. I do still think it's less confusing. Maybe the use of the full phrase "Candidate Recommendation Draft" in the next sentence would be enough to make that clear. Maybe we can even move the link (https://www.w3.org/TR/wasm-core-2/) to be anchored there rather than up here to be even more clear.